web-common/src/features/models/workspace/inspector/ModelInspectorModelProfile.svelte

```diff
@@ -102,35 +136,61 @@
           class="mt-1"
         >
           {#if viableSources && $viableSources}
-            {#each $viableSources as table (table.name)}
+            <EmbeddedSourceReferences
+              {references}
+              entries={viableEmbeddedSources}
+            />
+            {#each viableExplicitSources as source (source.name)}
               <WithModelResultTooltip {modelHasError}>
-                <div
-                  class="grid justify-between gap-x-2 {classes.QUERY_REFERENCE_TRIGGER} p-1 pl-4 pr-4"
+                <a
+                  href="/source/{source.name}"
+                  class="ui-copy-muted grid justify-between gap-x-2 {classes.QUERY_REFERENCE_TRIGGER} p-1 pl-4 pr-4"
                   style:grid-template-columns="auto max-content"
-                  on:focus={focus(table)}
-                  on:mouseover={focus(table)}
+                  on:focus={focus(source)}
+                  on:mouseover={focus(source)}
                   on:mouseleave={blur}
                   on:blur={blur}
                   class:text-gray-500={modelHasError}
                 >
-                  <div class="text-ellipsis overflow-hidden whitespace-nowrap">
-                    {table.name}
+                  <div
+                    class="text-ellipsis overflow-hidden whitespace-nowrap flex items-center gap-x-2"
+                  >
+                    <div
+                      class=" text-ellipsis overflow-hidden whitespace-nowrap"
```

Comment:
Nit: replace `text-ellipsis overflow-hidden whitespace-nowrap` here and above with `truncate`
https://tailwindcss.com/docs/text-overflow#truncate